func/python/pickle_check.py

```diff
@@ -0,0 +1,19 @@
+import pickle
```

Comment:
`pickle` is another `lib-dynload` library (i.e. uses a shared object under the hood), so it's important to test.